jcl/src/java.base/share/classes/java/lang/invoke/LambdaForm.java

```diff
@@ -59,6 +59,12 @@
 		MethodHandle resolvedHandle() {
 			throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
 		}
+
+		/*[IF Sidecar18-SE-OpenJ9]*/
+		MethodHandle resolve() {
```

Comment:
This wasn't previously needed: perhaps the condition ought to be `! Sidecar19-SE-OpenJ9`?

Reply:
A positive condition is better than a negative one, as the negative condition could end up including the code in unexpected places.